Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -1513,30 +1513,9 @@ private async Task<string> GetAuthorizationToken(User user, string deviceId, str
                     Limit = 1
```

Comment:
What if the user somehow has multiple sessions on the same device? Shouldn't we remove the duplicates?

Reply:
How likely is that given the check that returns the same session when the same user authenticated on the same device?

Reply:
It's probably very unlikely.

Another thing: If this flow is only triggered when the user attempts to log in, shouldn't we reset the token?

Reply:
I think we should remove _all_ of the users sessions on that device and create a new one. I don't really understand why the token is just reused.

Reply:
Agreed

Reply:
Update to remove sessions for the same user and device. I also tested that I could use access tokens for different users without needing to re-auth (although web doesn't currently support this so I had to change local storage values directly).